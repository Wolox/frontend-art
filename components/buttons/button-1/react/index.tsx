import React from 'react';

import '../styles.scss';

function Button1() {
  return (
    <div className="column center btn-group-1">
      <button className="btn btn-1 m-bottom-2" disabled={true}>
        <div className="btn__content">cancel</div>
      </button>
      <button type="button" className="btn btn-1">
        <div className="btn__content">submit</div>
      </button>
    </div>
  );
}

export default Button1;
