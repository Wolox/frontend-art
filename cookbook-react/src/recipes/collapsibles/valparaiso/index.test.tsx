import React from 'react';
import { render } from '@testing-library/react';

import Example from '.';

test('Should match snapshot', () => {
  const { container } = render(<Example />);
  expect(container).toMatchSnapshot();
});
