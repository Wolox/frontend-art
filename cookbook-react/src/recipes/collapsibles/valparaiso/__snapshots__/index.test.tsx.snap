// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<div>
  <div
    class="collapsibleContainer column"
  >
    <div
      class="row space-between m-bottom-2"
    >
      <h3
        class="title"
      >
        TÃ­tulo
      </h3>
      <button
        aria-controls="collapsable-example"
        aria-expanded="true"
        class="icon"
        type="button"
      />
    </div>
    <div
      class="content"
      id="collapsable-example"
      role="region"
      style="height: 0px;"
    >
      <p
        class="m-bottom-3 text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
      <button
        class="button"
        type="button"
      >
        Action
      </button>
    </div>
  </div>
</div>
`;
